src/http-spec.ts

```diff
@@ -9,6 +9,20 @@ import { IServer } from './servers';
 
 export interface IHttpService extends INode {
```

Comment:
@XVincentX can we add description on here too?

Reply:
Yes, it is inherited with `tags`